sdk/automation/Azure.ResourceManager.Automation/src/Generated/Models/AutomationActivityParameter.cs

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Automation.Models
+{
+    /// <summary>
+    /// Definition of the activity parameter.
+    /// Serialized Name: ActivityParameter
+    /// </summary>
+    public partial class AutomationActivityParameter
```

Comment:
Maybe we should rename it to `AutomationActivityParameterDefinition`.

Reply:
fixed